packages/solid-js/src/createFireproof.tsx

```diff
@@ -191,9 +191,14 @@ export function createFireproof(dbOrName?: string | Database, config: ConfigOpts
 
     createEffect(() => {
       const db = database();
-      const unsubscribe = db.subscribe(() => void refreshRows(db));
+      console.log("subscribing");
+      const unsubscribe = db.subscribe(() => {
+        console.log("subscription event");
+        void refreshRows(db)
+      }, true);
```

Comment:
you don't need this true unless you want to consume the updates. I'd leave it false for now, but setting it true was my first hint the subscription wasnt getting set